kmod/pkg.yaml

```diff
@@ -22,6 +22,7 @@ steps:
       - |
         cd build
         make DESTDIR=/rootfs install
+        for target in depmod insmod modinfo modprobe rmmod; do ln -s ../bin/kmod /rootfs${TOOLCHAIN}/bin/$target; done
```

Comment:
what's this for?

Reply:
it was there before my changes, 32 version creates these symlinks on its own, so I removed it